### PR TITLE
Changes I did apply and make it work. and improve Documentation

### PR DESCRIPTION
**Pull request by wrenkredhat**
Hello Accoring to my mail this is current state of play :-)

### CODE REVIEW

File: playbooks/roles/inventory/templates/hosts.j2 [hunk 1]
```diff
@@ -117,7 +136,7 @@ openshift_metrics_hawkular_hostname=metrics.{{_apps_dns}}
 openshift_metrics_cassandra_nodeselector={"node-role.kubernetes.io/infra": "true"}
 openshift_metrics_hawkular_nodeselector={"node-role.kubernetes.io/infra": "true"}
 openshift_metrics_heapster_nodeselector={"node-role.kubernetes.io/infra": "true"}
-
+b
```
Comment (pcfe) [inline playbooks/roles/inventory/templates/hosts.j2]: typo or is that lonely 'b' really supposed to be added?

File: playbooks/vars/guests.yml [hunk 2]
```diff
@@ -17,7 +17,7 @@ guests:
   network: bridge=virbr0
   os_type: linux
   os_variant: rhel7.6
-  disk_os_size: 40g
+  disk_os_size: 500g
```
Comment (pcfe) [inline playbooks/vars/guests.yml]: straight from 40 to 500? What are you putting on the OS disk?

File: playbooks/roles/inventory/templates/hosts.j2 (hunk 1 above)
Comment (wrenkredhat) [reply playbooks/roles/inventory/templates/hosts.j2]: no, this accidently was added when navigating with the keybord mouse.
its gone now.

**Comment by wrenkredhat** [comment]
Hello, just another PR, --- consisteny merged -- and FilesSystems set to 250g per node.

File: playbooks/vars/guests.yml (hunk 2 above)
Comment (wrenkredhat) [reply playbooks/vars/guests.yml]: ... The sort answer is: id did not work work with just 40g.
and the big structural change is starting glusterfs on ( master, infra, node ).
and the  mount /dev/vdb is reserved for glusterfs -- so i **assume** that the System and glusterfs share the FS on each node.
did Request advice per mail.

**Review by tahonen — APPROVED** [review]
Looks good

**Pull request merged**